src/components/Home.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+
+    <head>
+
+        
+
+    </head>
+
+    <body>
+
+        <!--
+            div holding the contents of the
+            side bar where links will be       
+        -->
```

Comment:
Should not have a sidebar class, but rather a component. Also Quasar, the CSS framework we're using has many built in tools & components for this exact purpose. Quasar layouts, page containers, and drawers handle basically everything you're trying to accomplish here as well as any dynamic resizing and fancy stuff, so I'd definitely read the documentation on that and play around with it. 

Check out some of these links:
- Quasar layout builder: https://quasar.dev/layout-builder
- Quasar frontend routing with layouts and pages: https://quasar.dev/layout/routing-with-layouts-and-pages
- Quasar documentation: https://quasar.dev/